<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeCertDlg</class>
 <widget class="QDialog" name="MakeCertDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make certificate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="17" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Issuer Param</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QLineEdit" name="mSubjectDNText"/>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="mCSRFileFindBtn">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mCSRFilePathText"/>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="mUseCSRFileCheck">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Use CSR File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="mSelfSignCheck">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>SelfSign</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_8">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Issuer Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="mIssuerNameCombo"/>
     </item>
    </layout>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Certificate policy</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="mAlgorithmText"/>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QLineEdit" name="mEmailText"/>
   </item>
   <item row="19" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mAlgorithmLabel">
     <property name="text">
      <string>Algorithm</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="mOptionLabel">
     <property name="text">
      <string>Key Param</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>UserName</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Issuer information</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QLineEdit" name="mSSNText"/>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Issuer Algorithm</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QComboBox" name="mProfileNameCombo"/>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Request Information</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="1">
    <widget class="QLineEdit" name="mIssuerAlgorithmText"/>
   </item>
   <item row="17" column="3">
    <widget class="QLineEdit" name="mIssuerOptionText"/>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="mUserNameText"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Request Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QComboBox" name="mReqNameCombo"/>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="mOptionText"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>SubjectDN</string>
     </property>
    </widget>
   </item>
   <item row="20" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>SSN</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Profile name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mReqNameCombo</tabstop>
  <tabstop>mAlgorithmText</tabstop>
  <tabstop>mOptionText</tabstop>
  <tabstop>mProfileNameCombo</tabstop>
  <tabstop>mUserNameText</tabstop>
  <tabstop>mSSNText</tabstop>
  <tabstop>mEmailText</tabstop>
  <tabstop>mSubjectDNText</tabstop>
  <tabstop>mIssuerAlgorithmText</tabstop>
  <tabstop>mIssuerOptionText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MakeCertDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MakeCertDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
